process/smartContract/process.go

```diff
@@ -230,6 +234,7 @@ func NewSmartContractProcessor(args ArgsNewSmartContractProcessor) (*scProcessor
 	log.Debug("smartContract/process: enable epoch for increment SCR nonce in multi transfer",
 		"epoch", sc.incrementSCRNonceInMultiTransferEnableEpoch)
 	log.Debug("smartContract/process: enable epoch for built in functions on metachain", "epoch", sc.builtInFunctionOnMetachainEnableEpoch)
+	log.Debug("smartContract/process: enable epoch for scr size invariant", "epoch", sc.scrSizeInvariantCheckEnableEpoch)
```

Comment:
`smartContract/process: enable epoch for scr size invariant check`

Reply:
done